server/controllers/templates/web_templates.go

```diff
@@ -407,6 +407,8 @@ var ProjectJobsTemplate = template.Must(template.New("blank.html.tmpl").Parse(`
     <script src="{{ .CleanedBasePath }}/static/js/xterm-4.9.0.js"></script>
     <script src="{{ .CleanedBasePath }}/static/js/xterm-addon-attach-0.6.0.js"></script>
     <script src="{{ .CleanedBasePath }}/static/js/xterm-addon-fit-0.4.0.js"></script>
+    <script src="{{ .CleanedBasePath }}/static/js/xterm-addon-search-0.7.0.js"></script>
```

Comment:
This seems like an older version

https://www.npmjs.com/package/xterm-addon-search

Can you verify which versions here are the latest?

Reply:
You need to use the correct versions for the xterm.js release. See https://github.com/xtermjs/xterm.js/releases/tag/4.9.0

Reply:
Ah so it seems like we need to bump xterm version then. Would it be OK to bump the versions here as part of this PR ?

If not to 5.x xterm, then at least 4.19.0 xterm which would then allow `xterm-addon-search@0.9.0`

https://github.com/xtermjs/xterm.js/releases/tag/4.19.0

Reply:
No, I tried that, it caused other issues.

Reply:
Ok. Thank you for trying. We'll have to upgrade at some point in another pr :(